### PR TITLE
patternfly - a new css

### PR DESCRIPTION
**Pull request by abenari**
For review only at the moment, this is a new set of css.

### CODE REVIEW

**Comment by abenari** [comment]
[test]

**Comment by dLobatog** [comment]
I've been testing this side by side with the old CSS and I can't find any differences, what's the reason behind this change?

**Comment by dLobatog** [comment]
Nevermind, I guess cached assets didn't allow me to see the changes.
- Font for buttons' text is remarkably small
- Host buttons are not colored and they are so small they are hard to notice in high resolutions
- Can we align 'add interface' buttons? 
- Edit template form fields are not aligned
- Submit buttons are very small
- Snippet button label is below and in a different color
- Note alerts (see template edit screenshot) have a different font size and more top-padding than it should 

See screenshots in this album for reference. http://imgur.com/a/uOiq9

**Comment by ehelms** [comment]
FYI, I am actively working on an equivalent update to Katello to account for the changes that come along with this update.

**Comment by domcleal** [comment]
Please ensure LICENSE is updated to indicate the licence of the new fonts, CSS files etc.

**Comment by abenari** [comment]
[test]

**Comment by ohadlevy** [comment]
relates to https://github.com/Katello/katello/pull/3898

File: LICENSE [hunk 1]
```diff
@@ -13,6 +16,7 @@ The Foreman repository/package is licensed as follows:
 * vendor/assets/javascripts/jquery.dataTables.min.js is GNU GPL v2/BSD
 * vendor/assets/javascripts/jquery.jeditable.js is MIT licensed
 * vendor/assets/javascripts/jquery.jnotify.js is ASL 2.0 licensed
+* vendor/assets/stylesheets/patternfly.scss is ASL 2.0 licensed
 
 Copyright holders for the Foreman project are in the separate file called
```
Comment (domcleal) [inline LICENSE:21]: Update this list too.

File: LICENSE [hunk 2]
```diff
@@ -5,7 +5,10 @@ The Foreman repository/package is licensed as follows:
 * config/locales/ and public/blank.html are MIT licensed
 * lib/tasks/convert.rake is under Revised BSD license
 * extras/noVNC/websockify is LGPL v3 licensed
-* vendor/assets/fonts/ is ASL 2.0 licensed
+* vendor/assets/fonts/overpass is ASL 2.0 licensed
+* vendor/assets/fonts/OpenSans is ASL 2.0 licensed
+* vendor/assets/fonts/PatternFlyIcons is ASL 2.0 licensed
+* vendor/assets/fonts/font-awesome is SIL OFL 1.1 licensed
```
Comment (domcleal) [inline LICENSE:11]: Include the text of the SIL OFL 1.1 licence in the lower section of this file.

**Comment by ohadlevy** [comment]
I did notice that alerts in patternfly usually have an additional icon indicator, e.g. as in https://www.patternfly.org/widgets/#alerts

just a red border does not seem enough in my taste?

**Comment by joshuabaird** [comment]
I have been reviewing the current design and the new Patternfly design side by side and here is some feedback:
- I'm partial to the additional padding in various elements that is present in the current design (especially in the tables and buttons).
  - I did notice that some additional padding is present in /fact_values, but not /hosts?  I would personally like to see this in all tables.
- I believe the button text should be slightly larger, although some additional padding (see #1) would help as well.
- I think h1 should be bolder.
- I like the new charts.
- Should we consider having a background color to the table header cells?  I think this would look nice on the /hosts page for example.

I'll keep playing with it, but overall I do think it's a pretty clean design and could be even better with a few tweaks.

**Comment by ehelms** [comment]
Any reason for closing this work?

**Comment by abenari** [comment]
I closed it unintentionally - reopening.

**Comment by domcleal** [comment]
Some random comments, haven't fully reviewed it.
1. Agreed with @joshuabaird about lack of padding throughout the app.  Facts seem to get away with it because the Chart button is large.
2. In Chrome 34.0.1847.132 on Linux, dropdown menus confusingly seem to drop up.
3. Text generally seems too small, particularly form input labels, form buttons, buttons on lists.
4. The size of buttons on lists seems disproportionately small compared to the menus and menu dropdown button on its right (see smart proxies list).
5. Status labels on host list are too small and have inconsistent sizes.
6. Left padding of header sections in application menus (e.g. Puppet under Configure) is out of alignment with other text, looks strange.  Text in the menus is also to the left of the top level menu text itself.
7. Tab colours on say the host edit look backwards to me, links to other tabs are no longer blue?

I'm not sure that I prefer it tbh.

**Comment by ehelms** [comment]
@abenari If you update the patternfly version you are using here, the issue with the base font-size should be resolved and address a lot of the sizing issues folks have.

**Comment by abenari** [comment]
@ehelms  the css is already up to date, base font size at 12px, see https://github.com/abenari/foreman/blob/46f19ffc493769888397938ec28a627ede129e77/vendor/assets/stylesheets/patternfly.scss#L1690

**Comment by iNecas** [comment]
/me likes the more compact tables with more info and less space more, as well as having the whole screen used for the data.

**Comment by ohadlevy** [comment]
closing, as no one is working on this atm

**Comment by dLobatog** [comment]
Do we want this at some point? I think it looks more professional than bootstrap, not sure if other people agree with me on that.

**Comment by mmoll** [comment]
I also like the enterprisy look I saw on Sat 6 :star2:

**Pull request closed without merging**